radio/src/gui/480x272/zone.h

```diff
@@ -0,0 +1,54 @@
+
+#ifndef _ZONE_H_
+#define _ZONE_H_
+
+#include <inttypes.h>
+#include <string.h>
+
+#define LEN_ZONE_OPTION_STRING         8
+
+#define OPTION_VALUE_UNSIGNED(x)    setZoneOptionValue((uint32_t)x)
```

Comment:
Yes... trying to reduce the usage... open to ideas  :)   Since that ZoneOptionValue union is used directly in some packed structs (eg. Theme data in RadioData) it kinda limits the options, seems like.